cmake/SetupCompilers.cmake

```diff
@@ -60,7 +60,7 @@ endif()
 # end RAJA_ENABLE_CUDA section
 
 if (RAJA_ENABLE_HIP)
-  set(RAJA_HIPCC_STD_FLAG -std=c++14)
+  set(RAJA_HIPCC_STD_FLAG -std=c++17)
```

Comment:
should the CUDA block be updated to 17 as well? its a few lines up.

Reply:
That's the only one I see as well. Amusing to see this during the first session of the meeting to finish out the last features before the freeze for C++26.